src/index.js

```diff
@@ -862,7 +862,7 @@ class MiniCssExtractPlugin {
                     'err.code = "CSS_CHUNK_LOAD_FAILED";',
                     "err.type = errorType;",
                     "err.request = realHref;",
-                    "linkTag.parentNode.removeChild(linkTag)",
+                    "if(linkTag.parentNode) linkTag.parentNode.removeChild(linkTag)",
```

Comment:
Please add space ` ` after `if`

Reply:
@alexander-akait I fixed it. Please review again~